### PR TITLE
Basic form components

### PR DESCRIPTION
**Pull request by inakijv**
Most important aspects:

- Basic form components added: Checkbox, radio buttons, input text, date & time and search.

screen shot 2017-04-10 at 4 45 46 pm

### CODE REVIEW

**Comment by inakijv** [comment]
ping @dpobel

File: ui-guidelines/css/components-form.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    -moz-box-sizing: border-box;
+    border: 1px solid #b1d2e0;
+    border-radius: 4px;
+    -webkit-transition: all 0.30s ease-in-out;
+    -moz-transition: all 0.30s ease-in-out;
+    -ms-transition: all 0.30s ease-in-out;
+    -o-transition: all 0.30s ease-in-out;
+    outline: none;  
+}
+input[type=text]:focus,
```
Comment (SylvainGuittard) [inline ui-guidelines/css/components-form.css]: Insert an empty line :)

File: ui-guidelines/css/components-form.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    letter-spacing: 0px;
+}
+
+.ez-form-inputtext-errorstyle label {
+    color: #d92d42;
+}
+
+.ez-form-inputtext-errorstyle label:after {
+    display: inline;
+    content: ' This field is required';
```
Comment (SylvainGuittard) [inline ui-guidelines/css/components-form.css]: Extra space before `This`?

**Comment by inakijv** [comment]
Reviewed & updated. Ping @SylvainGuittard

File: ui-guidelines/css/components-form.css [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+input[type=text], 
+input[type=date],
+input[type=datetime],
+input[type=number],
+input[type=search],
+input[type=time],
+input[type=url],
+input[type=email],
+textarea, 
+select{
```
Comment (SylvainGuittard) [inline ui-guidelines/css/components-form.css]: CS: space before `{`
Sorry I missed this

File: ui-guidelines/css/components-form.css [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+input[type=text]:focus, 
+input[type=date]:focus,
+input[type=datetime]:focus,
+input[type=number]:focus,
+input[type=search]:focus,
+input[type=time]:focus,
+input[type=url]:focus,
+input[type=email]:focus,
+textarea:focus, 
+select:focus{
```
Comment (SylvainGuittard) [inline ui-guidelines/css/components-form.css]: CS: space before `{`

File: ui-guidelines/css/components-form.css (hunk 3 above)
Comment (inakijv) [inline ui-guidelines/css/components-form.css]: Thanks!

File: ui-guidelines/css/components-form.css (hunk 4 above)
Comment (inakijv) [inline ui-guidelines/css/components-form.css]: Thanks!

**Pull request merged**